docusaurus/docs/components/table/index.md

```diff
@@ -569,6 +712,73 @@ const columns = [
 
 Note that this is not supported in IE 11.
 
+## Accessing the Table Ref in a Parent Component
+
+It is common that a parent component will need to have access to the the table instance. This can be done utilizing React Refs.
+
+There is now an exported `TableRef` that contains the react-table `tableInstance`.
+
+```jsx
+import React, { useRef } from 'react';
+import Table, { TableRef } from '@availity/table';
+
+import MyObject from '@types/MyObject';
+
+type Props = {
+  data: MyObject[];
+};
+
+const MyComponent = ({ data }: Props) : JSX.Element => {
+  const ref = useRef<TableRef<MyObject>>(null);
+  const [areAllSelected, setAreAllSelected] = useState(false);
+
+  const columns =  useMemo(
+    () =>
+    [
+      {
+        Header: 'Column 1',
+        accessor: 'column1',
+      },
+      {
+        Header: 'Column 2',
+        accessor: 'column2',
+      },
+      {
+        Header: 'Column 3',
+        accessor: 'column3',
+      },
+    ],
+    []
+  );
+
+
+  const toggleSelectAll = () => {
+    const selectAll = !areAllSelected;
+    setAreAllSelected(selectAll);
+    tableRef.current?.instance?.toggleAllRowsSelected(showAll);
+  };
+
+  return (
+    <>
+      <Input
+        id="checkSelectAll"
+        aria-labelledby="lblSelectAll"
+        type="checkbox"
+        checked={areAllSelected}
+        onChange={toggleSelectAll}
+      />
+      <Label id="lblSelectAll" className="ml-1 mr-2" check>
+      Select All
+      </Label>
+   <Table ref={ref} selectable columns={columns} data={data}/>
+  </>
```

Comment:
**NITPICK:**
```suggestion
        Select All
      </Label>
      <Table ref={ref} selectable columns={columns} data={data} />
    </>
```

Reply:
My auto-formatter isn't working for this file for whatever reason.